asserts/snap_asserts.go

```diff
@@ -99,9 +99,101 @@ func buildSnapBuild(assert assertionBase) (Assertion, error) {
 	}, nil
 }
 
+// SnapRevision holds a snap-revision assertion, asserting the properties of a
+// snap release by a store authority.
+type SnapRevision struct {
+	assertionBase
+	sequence  uint64
+	timestamp time.Time
+}
+
+// SnapID returns the snap id of the built snap.
+func (assert *SnapRevision) SnapID() string {
+	return assert.Header("snap-id")
+}
+
+// SnapDigest returns the digest of the built snap.
```

Comment:
yes,  something like sha256 (or sha3 as the security team recommended) and the base64 (maybe url safe) of the digest